### DISCUSSION TITLE
OptRunoffSurface default or recommended option changed in Noah-MP-5.0?

### DISCUSSION THREAD

**Discussion by dmocko** [NCAR/noahmp#138] [Q&A]
For Noah-MP-4.0.1, the recommended option for runoff/groundwater was "1":

```
! =====================================options for different schemes================================
! **recommended
```

```
  INTEGER :: OPT_RUN  ! options for runoff and groundwater
                      ! **1 -> TOPMODEL with groundwater (Niu et al. 2007 JGR) ;
                      !   2 -> TOPMODEL with an equilibrium water table (Niu et al. 2005 JGR) ;
                      !   3 -> original surface and subsurface runoff (free drainage)
                      !   4 -> BATS surface and subsurface runoff (free drainage)
                      !   5 -> Miguez-Macho&Fan groundwater scheme (Miguez-Macho et al. 2007 JGR; Fan et al. 2007 JGR)
                      !          (needs further testing for public use)
```

For Noah-MP-5.0, however, the various options are given as "default" instead of "recommended".

Further, on page 147 of the Noah-MP v5 TechNote, as well as in the code here:

https://github.com/NCAR/noahmp/blob/master/src/ConfigVarType.F90#L74-L82

... option "3" is instead listed as the "default".

Is option "1" no longer recommended?  What is the reason for the change for this option for v5?

**Comment by cenlinhe** [comment]
The option 3 is set to "default" because the National Water Model operationally use option 3 which shows a good performance. Also, option 1 typically requires a very long spin up time (e.g., more than 10 years) to allow the water table to be stable, and users typically do not have the capacity to do the long spin up. We are currently doing a global runoff scheme assessment for all the Noah-MP runoff options (to be submitted soon), which shows an overall good performance of using option 3 among all the options even outside CONUS.
